vassal-app/src/main/resources/VASSAL/i18n/VASSAL_ja.properties

```diff
@@ -522,6 +522,8 @@ PlayerRoster.become_observer=見学者になる
 PlayerRoster.join_another_side=別な陣営で参加
 PlayerRoster.give_up_position=%1$s の立場を放棄しますか？
 PlayerRoster.join_game_as=どの陣営で参加しますか？
+PlayerRoster.switch_sides=Your current side is %1$s. Switch to which side?
+PlayerRoster.switch_sides2=%1$s is now unavailable. Your current side is %2$s. Switch to which side?
```

Comment:
You definitely don't need to do this. The base language version will always be displayed as a fall-back if no language-specific version exists for that key.